roles/migrationcontroller/tasks/main.yml

```diff
@@ -492,5 +526,26 @@
       namespace: "{{ meta.namespace }}"
       status:
         phase: Failed
-    when: olm_managed and not(reconciled)
+    when: not(reconciled)
```

Comment:
Same here:
At least some versions of OCP3 don't have status sub-resources. We're doing this as a hacky way to allow the operator to work on OCP3 (back as far as 3.7) so we should leave it.